wolfssl/wolfcrypt/ed25519.h

```diff
@@ -47,6 +47,7 @@
     extern "C" {
 #endif
 
+#define FreeEd25519Key wc_ed25519_free
```

Comment:
What is this macro? Why not call `wc_ed25519_free` directly? If its for openssl compatibility it needs to be in the right wolfssl/openssl/ header.

Reply:
I followed the code style from dsa/ecc/rsa evp & ssl layer

Reply:
I'm also not sure what you are referring to with the code style of defining a function instead of using it directly?